### PR TITLE
feat: Sending Animation for FAB Button

### PR DESCRIPTION
**Pull request by adityastic**
Changes: 
- Performing animation with the fab button while sending data

Screenshots for the change:
animsend

### CODE REVIEW

**Comment by adityastic** [comment]
@anhanh11001 @yashk2000 Please Review

File: app/src/main/java/com/nilhcem/blenamebadge/ui/main/MainActivity.kt [hunk 1]
```diff
@@ -66,12 +65,14 @@ class MainActivity : AppCompatActivity(), PreviewChangeListener {
         fab_main.setOnClickListener {
             if (BluetoothAdapter.getDefaultAdapter().isEnabled) {
                 // Easter egg
-                fab_main.visibility = View.GONE
+                Toast.makeText(this, "Sending Data.", Toast.LENGTH_LONG).show()
```
Comment (anhanh11001) [inline app/src/main/java/com/nilhcem/blenamebadge/ui/main/MainActivity.kt]: I would extract the string in here. Rest looks good, the animation is really cool

Comment (yashk2000) [reply app/src/main/java/com/nilhcem/blenamebadge/ui/main/MainActivity.kt]: Use string resources instead of hardcoding the string.

Comment (adityastic) [reply app/src/main/java/com/nilhcem/blenamebadge/ui/main/MainActivity.kt]: done. have a look

**Comment by adityastic** [comment]
@iamareebjamal please review

**Pull request merged**